src/pipeline.py

```diff
@@ -40,10 +40,11 @@
 from nv_ingest.stages.transforms.image_caption_extraction import generate_caption_extraction_stage
 from nv_ingest.util.converters.containers import merge_dict
 from nv_ingest.util.logging.configuration import LogLevel
+from nv_ingest.util.logging.configuration import configure_logging as configure_local_logging
 from nv_ingest.util.schema.schema_validator import validate_schema
 
 logger = logging.getLogger(__name__)
-logger.setLevel(logging.INFO)
+configure_local_logging(logger, os.getenv("INGEST_LOG_LEVEL", "INFO"))
```

Comment:
Should not be done here, this is handled here: https://github.com/NVIDIA/nv-ingest/pull/14/files#diff-a5b564197d741ea776ee29ab7192991961c613741edc265c9c623c58c0a623cdL657

Reply:
It was previously hardcoded to logging.INFO and no debug logs were produced from the container when setting log_level in this file